app.py

```diff
@@ -145,4 +145,4 @@ def index():
 
 
 if __name__ == '__main__':
-    app.run(debug=True)
+    app.run(debug=True, port=9191)
```

Comment:
Two points:
- why there are two commits with the same purpose?
- why don't you change port number in nginx/uwsgi files? As the owner already merged previous PR, I think he wanted to use default port. Anyway, this is my opinion, owner decides all.

happy programming!